### PR TITLE
PT-1205 Display data based on timezone objects

### PR DESCRIPTION
**Pull request by yvon-dblg**
With https://github.com/mdblp/viz/pull/52

### CODE REVIEW

File: js/data/util/constants.js
Comment (BigFab) [inline js/data/util/constants.js]:
```suggestion
 * Used to regroup device parameters in one tooltip, when the changes are too close.
 * This is to avoid superpositions of the icons in the daily view.
```

File: package.json
Comment (BigFab) [inline package.json:22]: did we not once say we want to restrict the audit level to moderate?
--audit-level=moderate

**Review by BigFab — APPROVED** [review]
Some general comments, which are not really viz or tideline but both. Not sure it is feasible and whether it makes sense or not. But :)
- as a user, I think I would prefer seeing my data with the information "summer time/winter time" instead of "UTC+x" or "UTC+x+1"
- for the time change at the top of the timeline, I would also prefer seeing something like "winter time change" rather then "time change"

I was able to test thoroughly. I think it is working pretty well. I just have some doubts concerning the fact that having a row of data (e.g. "cbg") set to "UTC" timzeone will make the "timechange" icons disappear. This might be something to discuss further.
Waiting for that I am giving my approval. Not sure anyone else would like to test this before we merge? @franck-fourel ?

File: package.json
Comment (yvon-dblg) [reply package.json:22]: Was just for me, to have an idea, not used yet.
anyway, as of today:
```
> npm audit --production --registry=https://registry.npmjs.org/


                       === npm audit security report ===

found 0 vulnerabilities
 in 113 scanned packages
```

**Pull request merged**
